### PR TITLE
feat: New spec to get satelltie empty url repositories

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

**Review by xiangce — APPROVED** [review]
Approved pending the approval of Spec

**Comment by psachin** [comment]
@huali027 Can you please rebase ?

**Pull request merged**
